pkgs/development/python-modules/blis/default.nix

```diff
@@ -45,10 +45,15 @@ buildPythonPackage rec {
     "blis"
   ];
 
+  passthru = {
+    skipBulkUpdate = true;
```

Comment:
Really needs to come with an explanation.

Reply:
added.